plugins/outputs/signalfx/signalfx.go

```diff
@@ -28,6 +29,9 @@ var sampleConfig = `
     ## SignalFx API Token
     APIToken = "my-secret-key" # required.
 
+    ## BatchSize
+    BatchSize = 1000
```

Comment:
Does telegraph automatically overwrite struct fields based on the config?

Reply:
yes